src/bundle/Resources/public/js/OnlineEditor/buttons/ez-btn-styleslistitem.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import AlloyEditor from 'alloyeditor';
+
+export default class EzBtnStylesListItem extends AlloyEditor.ButtonStylesListItem {
+    /**
+     * Lifecycle. Renders the UI of the button.
+     *
+     * @instance
+     * @memberof ButtonStylesListItem
+     * @method render
+     * @return {Object} The content which should be rendered.
+     */
+    render() {
+        const className = this.props.name === this.props.activeStyle ? 'ae-toolbar-element active' : 'ae-toolbar-element';
```

Comment:
```suggestion
        let className = 'ae-toolbar-element';
        if (this.props.name === this.props.activeStyle) {
          className += ' active'
         } 
```
so you won't repeat basic class name

Reply:
This code is copied from the AE, let's keep it as it is.